src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final String name = "circle";
+    private int radius;
+
+    public Circle(int radius) {
+        setRadius(radius);
+    }
+
+    public Circle(int radius, Colors color) {
+        setRadius(radius);
+        setColor(color);
+    }

```

Comment:
```suggestion
    public Circle(int radius, Colors color) {
        super(color);
        this.radius = radius;
    }
```
let's have such a constructor in all child figure classes
and remove all other constructors